engine/api/src/main/scala/pl/touk/nussknacker/engine/api/context/ProcessCompilationError.scala

```diff
@@ -180,12 +180,19 @@ object ProcessCompilationError {
       InvalidPropertyFixedValue(paramName, label, value, values, nodeId.id)
   }
 
-  case class OverwrittenVariable(variableName: String, nodeId: String)
+  case class OverwrittenVariable(variableName: String, nodeId: String, paramName: Option[String])
```

Comment:
imo fieldName would be better,

also I'm wondering if we shouldn't have a separate PartSubGraphCompilationError for a case when we override field names, like OverwrittenFieldName

Reply:
also I'm not sure if making paramName (aka fieldName) implicit is a good idea
this is not a sort of dependency that you pass over?
wdyf?